config/3.1/README.adoc

```diff
@@ -0,0 +1,44 @@
+---
+title: "MicroProfile Config 3.1 (under development)"
+date: 2023-06-05
+summary: 
+---
+MicroProfile Config aims to provide a set of Java interfaces and programming models which allow Java developers to configure their cloud native applications. 
+The goal of this release is to improve the TCKs so that they can work well with CDI Lite.
```

Comment:
I'd change this to  "The release includes:".

As it stands, the TCK verbiage is in this sentence and duplicated in the first bullet.

Reply:
done